files/contaodemo/theme/src/scss/components/_slider.scss

```diff
@@ -1,116 +1,128 @@
 $slider-breakpoint: lg;
-.slider {
+
+.swiper {
   $parent: &;
 
-  &-control {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    gap: 1rem;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 1rem;
 
-    margin-block-start: 1rem;
+  #{$parent}-wrapper {
+    flex-basis: 100%;
+  }
 
-    height: 3rem;
+  #{$parent}-pagination {
+    position: inherit;
 
-    // fullsize on screens < 1024
-    .header-image & {
-      @include breakpoint(lg, max) {
-        padding-inline: $s-grid-outer-gutter--large;
-      }
-    }
-
-    #{$parent}-menu,
-    a {
-      position: static;
-      top: 0;
-    }
+    // Reset default SwiperJS styles
+    bottom: unset;
+    order: 2;
+    width: auto;
+    margin-top: unset;
```

Comment:
```suggestion
    margin-top: unset;
    line-height: 0;
```

There is still that one little line-height issue that can be corrected resetting it to 0 to match the current styles

*Before*
![image](https://github.com/user-attachments/assets/54c9d452-38d7-4ab8-ae9e-24308300b282)

*After*
![image](https://github.com/user-attachments/assets/7aeefaf9-4389-48db-8d50-a60b4b791da9)